app/pages/project/affinity/AffinityPage.tsx

```diff
@@ -95,12 +80,16 @@ const staticCols = [
     header: 'members',
     cell: (info) => <AffinityGroupMembersCell antiAffinityGroup={info.getValue()} />,
   }),
+  colHelper.accessor('id', Columns.id),
```

Comment:
I don't think having the ID column here and in almost no other table makes sense. The ID is on the detail page and can be copied in the row actions. I think we should address this everywhere at once later. This is #1834.